pinc/unicode.inc

```diff
@@ -96,14 +96,28 @@ function utf8_character_name($characters)
     return implode(" + ", $title_pieces);
 }
 
+function split_graphemes($string)
+{
+    $next = 0;
+    $maxbytes = strlen($string);
+
+    while ($next < $maxbytes)
+    {
+        $char = grapheme_extract($string, 1, GRAPHEME_EXTR_COUNT, $next, $next);
+        if (isset($char) && strlen($char) > 0)
```

Comment:
shouldn't it be just

`yield grapheme_extract($string, 1, GRAPHEME_EXTR_COUNT, $next, $next);`